src/main/java/qna/answer/Answer.java

```diff
@@ -0,0 +1,106 @@
+package qna.answer;
+
+import qna.CannotDeleteException;
+import qna.action.NullCheckAction;
+import qna.domain.DateTimeEntity;
+import qna.question.Question;
+import qna.user.User;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "answer")
+public class Answer extends DateTimeEntity implements NullCheckAction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "writer_id", foreignKey = @ForeignKey(name = "fk_answer_writer"))
+    private User user;
+
+    @Lob
+    @Column(name = "contents")
+    private String contents;
+
+    @Column(name = "delete", nullable = false)
+    private boolean deleted = false;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
+    private Question question;
+
+    public Answer(final User user, final Question question, final String contents) {
+        this(null, user, question, contents);
+    }
+
+    public Answer(final Long id, final User user, final Question question, final String contents) {
+        throwExceptionIsNullObject(user);
+        throwExceptionIsNullObject(question);
+        this.id = id;
+        this.user = user;
+        this.question = question;
+        this.contents = contents;
+
+        question.addAnswer(this);
+    }
+
+    protected Answer() {
+    }
+
+    public void throwExceptionNotDeletableUser(final User loginUser) throws CannotDeleteException {
+        if (!this.user.equals(loginUser)) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public Question getQuestion() {
+        return question;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public void deleteAnswer() {
```

Comment:
```suggestion
    public void delete() {
```
Answer 객체이니 메서드명에 반복해서 표시하지 않아도 되겠네요